bc_obps/service/handle_addresses_service.py

```diff
@@ -0,0 +1,38 @@
+from registration.api.utils.operator_utils import AddressesData
+from registration.models import Address
+
+
+class HandleAddressesService:
```

Comment:
I think we could use a more descriptive name than "Handle".
Maybe an `AddressesService` class with a `upsert_addresses_from_data(...)` method?

I would also add a bit of documentation, the phsical-, mailing- and prefix parameters are pretty confusing